src/l10n.js

```diff
@@ -3,36 +3,39 @@ import { getGettextBuilder } from '@nextcloud/l10n/dist/gettext.js'
 const gtBuilder = getGettextBuilder()
 	.detectLocale()
 
-// Decompress Translations to gettext format and add to gtBuilder
-TRANSLATIONS.forEach((lang) => {
-	const translations = {}
+// eslint-disable-next-line no-lone-blocks
+{
```

Comment:
I'm curious why you added this lone block. Does it have any effect? I don't see any scoped variables in there. They are all in the `TRANSLATIONS.forEach` loop anyway.

Is it related to the last example with the source type module here: https://eslint.org/docs/latest/rules/no-lone-blocks ?

Reply:
Just to avoid some inconsistencies when replace this entry with an array.

My idea is remove part of this code and avoid some unnecessary computations.